modules/oe_theme_content_news/oe_theme_content_news.module

```diff
@@ -29,3 +32,79 @@ function oe_theme_content_news_preprocess_node__oe_news(&$variables) {
     $variables['content']['featured_image']['#attributes']['class'][] = 'ecl-paragraph';
   }
 }
+
+/**
+ * Implements hook_preprocess_HOOK() for node--oe-news--teaser.html.twig.
+ */
+function oe_theme_content_news_preprocess_node__oe_news__teaser(&$variables) {
+  /** @var \Drupal\node\NodeInterface $node */
+  $node = $variables['node'];
+
+  $cache = CacheableMetadata::createFromRenderArray($variables);
+
+  // Use the short title if present, with fallback to the title.
+  $variables['title'] = $node->get('oe_content_short_title')->value ?? $node->label();
+
+  $date_formatter = \Drupal::service('date.formatter');
+  $variables['meta'][] = $date_formatter->format($node->get('oe_news_publication_date')->value, 'oe_theme_news_date');
+
+  $media = $node->get('oe_news_featured_media')->entity;
```

Comment:
Let's first check `$node->get('oe_news_featured_media')->isEmpty()` before fetching the entity.